<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CommentWidget</class>
 <widget class="QWidget" name="CommentWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>537</width>
    <height>683</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QWidget" name="verticalLayoutWidget_2">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>30</y>
     <width>481</width>
     <height>271</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="field_lay"/>
  </widget>
  <widget class="QWidget" name="">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>330</y>
     <width>258</width>
     <height>280</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="main_lay">
    <item row="3" column="1">
     <widget class="QLabel" name="label">
      <property name="text">
       <string>نقاط قوت</string>
      </property>
     </widget>
    </item>
    <item row="5" column="0">
     <widget class="QLineEdit" name="le_disadv"/>
    </item>
    <item row="3" column="0">
     <widget class="QLineEdit" name="le_adv"/>
    </item>
    <item row="5" column="1">
     <widget class="QLabel" name="label_2">
      <property name="text">
       <string>نقاط ضعف</string>
      </property>
     </widget>
    </item>
    <item row="0" column="0" colspan="3">
     <widget class="QTextEdit" name="txt_content">
      <property name="styleSheet">
       <string notr="true">font: 10pt &quot;MS Shell Dlg 2&quot;;</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>570</y>
     <width>194</width>
     <height>64</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="1">
     <widget class="QPushButton" name="bnt_cancel">
      <property name="styleSheet">
       <string notr="true">background-color: rgb(241, 0, 0);
color: rgb(255, 255, 255);</string>
      </property>
      <property name="text">
       <string>انصراف</string>
      </property>
     </widget>
    </item>
    <item row="0" column="0">
     <widget class="QPushButton" name="bnt_save">
      <property name="styleSheet">
       <string notr="true">background-color: rgb(0, 239, 0);
color: rgb(255, 255, 255);</string>
      </property>
      <property name="text">
       <string>ثبت نظر</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
